### PR TITLE
Added C128 accelerator code and documentation for both C128 mode and C128 in C64 mode.

### PR DESCRIPTION
**Pull request by blackystardust**
Added C128 in C64 mode accelerator code and documentation.

### CODE REVIEW

**Comment by greg-king5** [comment]
This PR doesn't make sense.  There's no difference between 128 mode and 64 mode -- as far as speed control is concerned.  There should be only a C128 acceleration PR.  It should work for both modes!
- The header should say that the C128 functions are for both 128 mode and 64 mode.
- Their library source code should be in `libsrc/c128/`.
- `libsrc/c64/` should have files with the same names; but, they simply should `.include` the c128 files.

**Comment by blackystardust** [comment]
This PR is for the C64.

For a C128 program the 8502 is always present, even though it might not be the CPU in control (in case of the SuperCPU), and you can (almost) count on being able to use the fast(), slow() and isfast() code, these 3 functions are c128 native only, when compiling a C64 program it doesn't make sense to assume that the program is running on a C128 in C64 mode and can therefor use the fast/slow functions.

So this code treats the C128 CPU as an accelerator for a C64 program.

File: libsrc/c64/acc_c128_speed.s [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+_set_c128_speed:
+        cmp     #SPEED_2X
+        bcs     high_speed
+store_speed:
+        sta     C128_VICIIE_CLK
+        jmp     _get_c128_speed
+
+high_speed:
+        lda     #$01
+        bne     store_speed
```
[the remark this answers is not in the document]
Comment (greg-king5) [reply libsrc/c64/acc_c128_speed.s]: I thought of even simpler code:
```asm
_set_c128_speed:
        cmp     #SPEED_2X
        lda     #$00                    ; clear VIC-IIe test bit
        rol     a                       ; carry flag is speed bit
        sta     C128_VICIIE_CLK

_get_c128_speed:
```

**Comment by oliverschmidt** [comment]
> My point is that it's foolish [...]

In many areas of the world one can easily see that the way people deal with each other gets rougher and rougher every day :-( I don't want to see this change in the cc65 community - even if you call it an Island of the Blissful.

Please note that my concern above is purely about style. It has nothing to do at all with the quality of the content.

**Comment by blackystardust** [comment]
My latest commit changes the code to work for both native c128 and c64 mode, and adjusted the documentation accordingly.

**Comment by oliverschmidt** [comment]
Sorry if I'm yet again slow on take up or missing the point altogether, but... As far as I can tell the latest commit to this pull request takes care of all concerns brought up so far. Please provide feedback if that assessment is correct.

**Comment by oliverschmidt** [comment]
As there's no other feedback (hopefully I didn't drive off Greg with my style remark) ...

**Pull request merged**